src/js/components/Select/SelectContainer.js

```diff
@@ -323,15 +309,15 @@ class SelectContainer extends Component {
               <SelectTextInput
                 focusIndicator={!customSearchInput}
                 size="small"
-                ref={this.searchRef}
+                ref={searchRef}
                 type="search"
-                value={search}
+                value={search || ''}
```

Comment:
Following our discussion of controlled/uncontrolled components, I thought we are trying to avoid setting initial values that will force components to be controlled by grommet.

Reply:
This is the embedded TextInput for searching inside the Select drop down. So, we want to control this one.